src/www/containers/userprofile/UserProfile.scss

```
@@ -0,0 +1,350 @@
+.Profile-page {
+  background-image: url("/img/profile/sky.jpg");
+  display: flex;
+  flex-direction: row;
+  padding: 4rem 8rem;
+
+  .Bio-panel {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    width: 15rem;
+
+    .Profile-photo {
+      border-radius: 0.5rem;
+      box-shadow: 1px 1px black;
+      height: 15rem;
+
+      .Edit-photo-button {
+        display: none;
+      }
+
+      &-edit {
+        .Edit-photo-button {
+          background-color: rgba($color: white, $alpha: 0.6);
+          padding: 0.5rem;
+          border: none;
+          cursor: pointer;
+          border-radius: 5px;
+        }
+      }
+
+      &,
+      &-edit {
+        .Photo {
+          border-radius: 0.5rem;
+          box-shadow: 1px 1px black;
+          height: 15rem;
+        }
+      }
+    }
+
+    .Username {
+      font-weight: bold;
+      font-size: 1.5rem;
+    }
+
+    .Info {
+      font-size: 1rem;
+      margin-top: -0.5rem;
+      max-width: 15rem;
+      overflow-wrap: break-word;
+
+      &-edit {
+        .Edit-bio-input {
+          display: flex;
+          resize: none;
+          height: 5rem;
+          border-radius: 0.3rem;
+          width: 98%;
+        }
+
+        .Save-button,
+        .Cancel-button {
+          align-items: center;
+          background-color: rgba($color: white, $alpha: 0.6);
+          border: 1px solid rgba($color: black, $alpha: 0.3);
+          border-radius: 0.3rem;
+          box-shadow: 0.5px 0.5px 0.5px 0.5px black;
+          cursor: pointer;
+          display: flex;
+          flex-direction: column;
+          font-size: 1rem;
+          margin-top: 0.5rem;
+          outline: none;
+          padding: 0.2rem;
+          transition: 0.15s ease-in-out;
+          width: 100%;
+        }
+
+        .Save-button {
+          &:hover {
+            background-color: rgba($color: lightgray, $alpha: 0.6);
```

What do you think about making this green to contrast the red one below?
```suggestion
            background-color: rgba($color: limegreen, $alpha: 0.6);
```